kuma/payments/utils.py

```diff
@@ -53,4 +53,5 @@ def cancel_stripe_customer_subscription(stripe_customer_id):
     customer = stripe.Customer.retrieve(stripe_customer_id)
     for sub in customer["subscriptions"]["data"]:
         s = stripe.Subscription.retrieve(sub["id"])
+        yield s
```

Comment:
See longer comment about why I made this change. I'm still not satisfied with this. I think what it ought to do instead is to trigger Django signals. Custom ones. One for subscriptions having been created and one for subscriptions about to be canceled. I don't mind working on refactoring like that but my PR-spidy senses tells me this PR is already large and complex enough. I'll file some followup issues.